src/Evolution/DiscontinuousGalerkin/Actions/NormalCovectorAndMagnitude.hpp

```diff
@@ -0,0 +1,165 @@
+// Distributed under the MIT License.
```

Comment:
This file doesn't define an action, so it's probably better placed one directory up

Reply:
It's a detail for one of the actions, so I think this is better than any other location.

Reply:
In that case put it in a `detail` namespace?

Reply:
I don't want to deal with more renames. Renaming things has already taking up a lot of my time. I don't really see the benefit here of changing things more until things are working.